Makefile

```diff
@@ -97,13 +97,19 @@ SED ?= $(shell which gsed 2>/dev/null || which sed)
 	@touch $@
 
 # This target compiles mimir for linux/amd64 and linux/arm64 and then builds and pushes a multiarch image to the target repository.
-# We don't separate building of single-platform and multiplatform images here (as we do for push-multiarch-build-image), as
-# Mimir's Dockerfile is not doing much, and is unlikely to fail.
-push-multiarch-mimir:
-	@echo
-	$(MAKE) GOOS=linux GOARCH=amd64 BINARY_SUFFIX=_linux_amd64 cmd/mimir/mimir
-	$(MAKE) GOOS=linux GOARCH=arm64 BINARY_SUFFIX=_linux_arm64 cmd/mimir/mimir
-	$(SUDO) docker buildx build -o type=registry --platform linux/amd64,linux/arm64 --build-arg=revision=$(GIT_REVISION) --build-arg=goproxyValue=$(GOPROXY_VALUE) --build-arg=USE_BINARY_SUFFIX=true -t $(IMAGE_PREFIX)mimir:$(IMAGE_TAG) cmd/mimir
+# We don't do separate building of single-platform and multiplatform images here (as we do for push-multiarch-build-image), as
+# these Dockerfiles are not doing much, and are unlikely to fail.
+push-multiarch-%/$(UPTODATE):
+	$(eval DIR := $(patsubst push-multiarch-%/$(UPTODATE),%,$@))
+	$(eval EXE := $(patsubst push-multiarch-%/$(UPTODATE),%,$@))
```

Comment:
What is EXE used for? Also looks equal to DIR value.

Reply:
Leftover from testing, thanks for catching that.